### DISCUSSION TITLE
What are the SimplePlanner profiles?

### DISCUSSION THREAD

**Discussion by rjoomen** [tesseract-robotics/tesseract_planning#382] [Q&A]
I was wondering what the SimplePlanner profiles exactly do. I couldn't find documentation, only [a suggestion by ](https://github.com/tesseract-robotics/tesseract_planning/issues/147#issuecomment-1003113833)@Levi-Armstrong somewhere to use the SimplePlannerFixedSizeAssignPlanProfile to solve a difficult planning problem.

simple_planner_fixed_size_assign_plan_profile.h
SimplePlannerFixedSizeAssignPlanProfile 

simple_planner_fixed_size_plan_profile.h
SimplePlannerFixedSizePlanProfile 

simple_planner_lvs_no_ik_plan_profile.h
SimplePlannerLVSNoIKPlanProfile 

simple_planner_lvs_plan_profile.h
SimplePlannerLVSPlanProfile 

simple_planner_profile.h
SimplePlannerPlanProfile

**Comment by Levi-Armstrong** [comment]
The simple planner is used to interpolate a input trajectory to create a new trajectory which meets the interpolation criteria defined in the profiles.
